python/knext/knext/builder/operator/builtin/deepke_prompt.py

```diff
@@ -0,0 +1,370 @@
+#
+# Copyright 2023 OpenSPG Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.
+
+import json
+import re
+from abc import ABC
+from typing import List, Dict, Tuple, Any
+from collections import defaultdict
+
+from knext.schema.client import SchemaClient
+from knext.schema.model.schema_helper import SPGTypeName, PropertyName, RelationName
+from knext.builder.operator.op import PromptOp
+from knext.builder.operator.spg_record import SPGRecord
+from knext.builder.operator.builtin.auto_prompt import AutoPrompt
+import uuid
+
+
+class DeepKEPrompt(AutoPrompt):
```

Comment:
Change DeepKE to the standardized name OneKE consistent with this month‘s release?